### PR TITLE
Updating to new faker library

### PR DESCRIPTION
**Pull request by funkymonkeymonk**
Updating dep.
Fixing issue #18

### CODE REVIEW

**Comment by funkymonkeymonk** [comment]
Ahh still had a working version of faker-factory in my path... Hoping into a virtual env to fix this.

**Comment by funkymonkeymonk** [comment]
Had to update the __init__.py to pass flake8. Also it looks like the new Faker may have issues on python 2.6 so I backed out the upgrade and pinned to the old version of faker-factory. Figured this way maintainers can decide when to upgrade and when not to.

**Comment by funkymonkeymonk** [comment]
Looks like it might not matter and you may want to just drop python 2.6 support anyway. I'll leave for maintainer to comment

File: robotframework-faker/FakerLibrary/__init__.py [hunk 1]
```diff
@@ -1,8 +1,6 @@
-__author__ = 'gkisel'
-
 import pkg_resources
-
 from keywords import FakerKeywords
+__author__ = 'gkisel'
```
Comment (guykisel) [inline robotframework-faker/FakerLibrary/__init__.py]: this can actually probably just be deleted, it's kinda pointless

File: setup.py [hunk 2]
```diff
@@ -33,7 +33,7 @@
     keywords=('robotframework testing '
               'test automation testautomation '
               'atdd bdd faker'),  # arbitrary keywords
-    install_requires=['fake-factory', 'robotframework', 'wrapt'],
+    install_requires=['fake-factory==0.7.4', 'robotframework', 'wrapt'],
```
Comment (guykisel) [inline setup.py]: i think we should just swap in `faker` - https://pypi.python.org/pypi/Faker

also, i'd rather not pin the dep to a specific version. it's friendlier to leave it unpinned and let developers who are installing this specify a specific version if they choose to.

**Comment by guykisel** [comment]
i'm okay with dropping 2.6 support. 2.7 has been out for an extremely long time.

**Comment by funkymonkeymonk** [comment]
Implemented. I also dropped the 2.6 check from travis to get the green build. We probably should have dropped the 2.6 from the setup.py as well.

**Comment by guykisel** [comment]
nice! if you could just drop the 2.6 from setup.py i'll be happy to merge this and create a new release :)

**Comment by guykisel** [comment]
actually, there are a few more 2.6-specific workarounds in the travis config file that could probably be removed/fixed as well. i can fix them myself after merge though.

**Comment by funkymonkeymonk** [comment]
Done

**Comment by guykisel** [comment]
thanks!

**Pull request merged**